sdk/java/pulumi/src/main/java/com/pulumi/deployment/InvokeOptions.java

```diff
@@ -17,7 +17,7 @@
  * Options to help control the behavior of @see {@link Deployment#invokeAsync(String, TypeShape, InvokeArgs, InvokeOptions)}.
  */
 @ParametersAreNonnullByDefault
-public final class InvokeOptions {
+public class InvokeOptions {
```

Comment:
I want `InvokeOutputOptions` to inherit from this, do we care about this being `final`?